delta/app/src/main/scala/ch/epfl/bluebrain/nexus/delta/wiring/MigrationModule.scala

```diff
@@ -76,6 +86,42 @@ class MigrationModule(appCfg: AppConfig, config: Config)(implicit classLoader: C
     HttpClient()(config.httpClient, as.classicSystem, sc)
   }
 
+  make[EventLog[Envelope[Event]]].fromEffect { databaseEventLog[Event](_, _) }
+
+  make[EventExchangeCollection].from { (exchanges: Set[EventExchange]) =>
+    EventExchangeCollection(exchanges)
+  }
+
+  make[ProjectsStatistics].fromEffect {
+    (
+        appCfg: AppConfig,
+        eventLog: EventLog[Envelope[Event]],
+        as: ActorSystem[Nothing],
+        sc: Scheduler
+    ) =>
+      implicit val system: ActorSystem[Nothing] = as
+      implicit val scheduler: Scheduler         = sc
+      val projection                            = appCfg.database.flavour match {
+        case Postgres  => Projection.postgres(appCfg.database.postgres, ProjectStatisticsCollection.empty)
+        case Cassandra => Projection.cassandra(appCfg.database.cassandra, ProjectStatisticsCollection.empty)
+      }
+      projection.flatMap { p =>
+        ProjectsStatistics(appCfg.projects, p, eventLog.eventsByTag(Event.eventTag, _))
+      }
+  }
+
+  make[ResolverScopeInitialization].from { (resolvers: Resolvers, serviceAccount: ServiceAccount) =>
```

Comment:
same here. If this is just used to power `ScopeInitialization` it can be done directly there with the .add clause